src/olympia/amo/middleware.py

```diff
@@ -324,3 +323,19 @@ def process_response(self, request, response):
             response['X-AMO-Request-ID'] = request.request_id
 
         return response
+
+
+class PinningRouterMiddleware(_PinningRouterMiddleware, MiddlewareMixin):
+    """Wrapper to allow old style Middleware to work with django 1.10+.
+    Will be unneeded once
+    https://github.com/jbalogh/django-multidb-router/pull/41 is merged and a
+    new release of django-multidb-router is available."""
+    pass
```

Comment:
Making a release with that PR merged right now :)

Reply:
dropped the wrapper in https://github.com/mozilla/addons-server/pull/9318/commits/5297c181f147c1bbe5836915875c32b6bfc339d2